modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -644,10 +759,10 @@ PERF_TEST_P_(CmpPerfTest, TestPerformance)
 
 PERF_TEST_P_(CmpWithScalarPerfTest, TestPerformance)
 {
-    MatType type    = -1;
-    CmpTypes opType = CMP_EQ;
-    cv::Size sz;
     compare_f cmpF;
+    CmpTypes opType;
+    cv::Size sz;
+    MatType type;
```

Comment:
Please add initialization with default values. See example in the `MulPerfTest` case (above).

Reply:
Applied, thanks